src/components/select/select.scss

```diff
@@ -0,0 +1,59 @@
+@import 'commons';
+
+.m-select {
+    display: inline-flex;
+    flex-direction: column;
+
+    &.m--has-validation-message {
+        .m-select__validation-message {
+            margin-top: $m-spacing--xs;
+        }
+    }
+
+    &:not(.m--is-disabled) {
+        .m-select__arrow {
+            cursor: pointer;
+        }
+    }
+
+    &__list {
+        height: auto;
+        overflow-x: hidden;
+        overflow-y: auto;
+        margin: 0;
+        padding: 0;
+        user-select: none;
+        list-style: none;
+        text-align: left;
+        background: $m-color--white;
+
+        @media (min-width: $m-mq--min--s) {
+            max-height: 208px;
```

Comment:
Ajouter un commentaire `// Magic number` qui justifira la hauteur
**Par exemple**
`// Magic number: Height of 4 items`